action/organization/user/list.go

```diff
@@ -37,5 +37,16 @@ func list(w http.ResponseWriter, r *http.Request) {
 		OrganizationID: uint(orgID),
 	}).Preload("User").Find(&users)
 
-	renderx.JSON(w, http.StatusOK, users)
+	result := []userWithPermission{}
```

Comment:
Taking this issue into consideration slice returns null value instead of empty slice so where ever slice is used then
declare it with **make(type, length)** ex: result = make([]userWithPermission, 0)